ballerina/soap_utils.bal

```diff
@@ -22,8 +22,9 @@ import ballerina/mime;
 # + soapAction - SOAP action
 # + body - SOAP request body as an `XML` or `mime:Entity[]` to work with soap attachments
 # + soapVersion - The SOAP version of the request
+# + headers - SOAP headers as a `map<string|string[]>`
 # + return - The SOAP Request sent as `http:Request`
-function createHttpRequest(SoapVersion soapVersion, xml|mime:Entity[] body, string soapAction)
+function createHttpRequest(SoapVersion soapVersion, xml|mime:Entity[] body, string soapAction, map<string|string[]>? headers = {})
```

Comment:
```suggestion
function createHttpRequest(SoapVersion soapVersion, xml|mime:Entity[] body, string soapAction, map<string|string[]> headers = {})
```
We can remove this also right?
Then we can remove the if condition as well in L47

Reply:
Fixed in [135bb19](https://github.com/ballerina-platform/module-ballerina-soap/pull/96/commits/135bb198b1b1f992c27d00ec2ac8eb639640abaf)